config/smtp.yml

```diff
@@ -6,6 +6,9 @@ development:
   enable_starttls_auto: <%= ENV['SMTP_ENABLE_STARTTLS_AUTO'] == 'true' ? true : false %>
   user_name: <%= ENV['SMTP_USER_NAME'] || "" %>
   password: <%= ENV['SMTP_PASSWORD'] || "" %>
+  openssl_verify_mode: <%= ENV['SMTP_OPENSSL_VERIFY_MODE'].presence || 'null' %>
```

Comment:
Yea, I think the default is nil and actually let's the SSL library decide.

On Sunday, September 13, 2015, level20peon <notifications@github.com> wrote:

> In config/smtp.yml
> <https://github.com/cantino/huginn/pull/1028#discussion_r39347910>:
>
> > @@ -6,6 +6,9 @@ development:
> >    enable_starttls_auto: <%= ENV['SMTP_ENABLE_STARTTLS_AUTO'] == 'true' ? true : false %>
> >    user_name: <%= ENV['SMTP_USER_NAME'] || "" %>
> >    password: <%= ENV['SMTP_PASSWORD'] || "" %>
> > +  openssl_verify_mode: <%= ENV['SMTP_OPENSSL_VERIFY_MODE'].presence || 'null' %>
>
> I couldn't find that default, so I assumed leaving it empty / null would
> be just as good. If peer indeed is the default, I surely would prefer to
> use that.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cantino/huginn/pull/1028/files#r39347910>.
>